templates/stretch/etc/fail2ban/jail.conf.erb

```diff
@@ -135,7 +135,7 @@ action_mwl = %(banaction)s[name=%(__name__)s, port="%(port)s", protocol="%(proto
 # Optionally you may override any other parameter (e.g. banaction,
 # action, port, logpath, etc) in that section within jail.local
 
-[ssh]
+[sshd]
```

Comment:
is it possible that it is named ssh  on some platforms and sshd on others?

Reply:
I don't think so, at least on Debian based packages upstream jail.conf is used and upstream uses [sshd].